src/client/components/DetailsModal/Details/components/DetailInput.tsx

```diff
@@ -155,6 +156,45 @@ export const AreaInput: React.FC<AreaInputProps> = ({
   />
 )
 
+type MileageInputProps = {
+  name: string
+  label: string
+  formikProps: FormikProps<QuoteInput>
+}
+
+export const MileageInput: React.FC<MileageInputProps> = ({
+  name,
+  label,
+  formikProps,
+}) => {
+  const { isoLocale } = useCurrentLocale()
+  const textKeys = useTextKeys()
+  const unitMultipler = isoLocale.toLowerCase().includes('en') ? 10 : 1
+  const options = [1000, 1500, 2000, 2500, 2501].map((value) => {
+    return value !== 2501
+      ? {
+          label: `${value *
+            unitMultipler} ${textKeys.CAR_INSURANCE_MILEAGE_UNIT()}`,
```

Comment:
Not sure how to accomplish this, but should ideally be:

```suggestion
            unitMultipler} ${textKeys.SE_WEB_SIGNUP_CAR_NEW_INSURANCE_MILEAGE_OPTION_{value}()}`,
```

since it needs to say "/year" as well, like "1 000 mil/år" or "10,000 km/year", or at least a translation with a template so that translators can place the value wherever they want: "{value} mil/år" and so on.

We have translations `SE_WEB_SIGNUP_CAR_NEW_INSURANCE_MILEAGE_OPTION_10000`, `SE_WEB_SIGNUP_CAR_NEW_INSURANCE_MILEAGE_OPTION_15000` and so on.